### PR TITLE
Add support for commodore templates

### PR DESCRIPTION
**Pull request by DebakelOrakel**
## Checklist

- [x] PR contains a single logical change (to build a better changelog).
- [ ] Update the documentation.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [ ] Link this PR to related issues or PRs.

### CODE REVIEW

**Review by simu — CHANGES_REQUESTED** [review]
Please fix the `.cruft.json` to match the actual component state (at least the list of test cases is broken). Also please delete the obsolete `.sync.yml`.

See https://syn.tools/syn/how-tos/prepare_for_component_sync.html for the recommended approach. tl;dr: run `commodore component new` with flags that match the component state and copy the resulting `.cruft.json`

**Comment by DebakelOrakel** [comment]
> Please fix the `.cruft.json` to match the actual component state (test cases and github org are wrong, copyright year shouldn't be updated either). Also please delete the obsolete `.sync.yml`.
> 
> See https://syn.tools/syn/how-tos/prepare_for_component_sync.html for the recommended approach. tl;dr: run `commodore component new` with flags that match the component state and copy the resulting `.cruft.json`

Yes, the update was broken. Should be fixed now.

File: .cruft.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      "slug": "openshift4-networking",
+      "parameter_key": "openshift4_networking",
+      "test_cases": "defaults syn-monitoring",
+      "add_lib": "n",
+      "add_pp": "n",
+      "add_golden": "y",
+      "add_matrix": "y",
+      "add_go_unit": "n",
+      "copyright_holder": "VSHN AG <info@vshn.ch>",
+      "copyright_year": "2023",
```
Comment (simu) [inline .cruft.json:17]: Please don't randomly change the copyright year

```suggestion
      "copyright_year": "2022",
```

File: .cruft.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "template": "https://github.com/appuio/commodore-component-template.git",
+  "commit": "3e1ac242633b3345621ba9f2b72f8dfea8e6ac4a",
+  "checkout": "main",
+  "context": {
+    "cookiecutter": {
+      "name": "Openshift4 Networking",
```
Comment (simu) [inline .cruft.json:7]: Usually, if we use a nice display name for OCP4 components, we use "OpenShift 4"

```suggestion
      "name": "OpenShift 4 Networking",
```

**Review by simu — APPROVED** [review]

File: .cruft.json (hunk 2 above)
Comment (DebakelOrakel) [reply .cruft.json]: Should that also be:
```
"checkout": "master",
```

Comment (DebakelOrakel) [reply .cruft.json]: Ah no, thats for the template.

**Pull request merged**